src/components/DialogHeader/DialogHeader.tsx

```diff
@@ -0,0 +1,70 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { ReactElement } from 'react';
+import { merge } from '@utilities/merge';
+import { IconCross } from '@foundation/Icon/Generated';
+import { Text } from '@typography/Text';
+
+export type DialogHeaderProps = {
+    title: string;
+    size?: DialogHeaderSize;
+    collapseBottom?: boolean;
+    onClose?: () => void;
+    icon?: ReactElement;
+    badge?: ReactElement;
+    'data-test-id'?: string;
+};
+
+export enum DialogHeaderSize {
```

Comment:
We were changing these enums to literal union types in other components... Maybe we should change here too?

Reply:
I discussed that with Jonas, Tomorrow I'm challenging it with the Fronti-FE community.